package.json

```diff
@@ -31,15 +35,24 @@
     "commander": "^2.20.0",
     "js-yaml": "^3.13.1",
     "jsonfile": "^5.0.0",
-    "winston": "^3.2.1"
+    "winston": "^3.2.1",
+    "webpack": "^4.41.2",
+    "webpack-merge": "^4.2.2",
+    "webpack-node-externals": "^1.7.2"
```

Comment:
These should be dev dependencies.

Reply:
@artlogic 
I agree with you, but eslint does not:
![image](https://user-images.githubusercontent.com/6169950/67288546-91907e80-f4b3-11e9-8823-250b40ad50d1.png)

Reply:
This needs to be configured - see https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-extraneous-dependencies.md#options